wordpress-mobile/gutenberg-mobile
# [iOS] Activate autoscroll on ListBlock

This PR updates the gutenberg ref to test https://github.com/WordPress/gutenberg/pull/15048

This activates autoscroll on iOS when the caret is out of the viewport area (i.e. goes behind the keyboard).

![lists-autoscroll](https://user-images.githubusercontent.com/9772967/56372799-fbc42000-61ff-11e9-8d9c-44fb6dfa7d89.gif)

To test:
- Create a list block.
- Add elements until the caret will go under the keyboard.
- Check that the block scrolls to show the caret.
